website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,188 @@
+
+![Showing the different list positions (left, right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
+
+By default, the width of the list matches the trigger width.
+
+![Showing the list's width matching the trigger's](/assets/components/form/super-select/super-select-list-width-matching.png)
+
+#### Auto width
+
+If the list width is set to “auto" it will match the width of the longest list item, which may result in the list being narrower or wider than the trigger width. A `max-width` can also be defined for the list.
+
+![Showing the list's auto width, resizing to the longest list item](/assets/components/form/super-select/super-select-list-width-auto.png)
+
+## ListItem Types
+
+There are four types of ListItems, two of which are component-specific.
+
+- `Checkmark` is used in the Single component to indicate a selection has been made.
+- `Checkbox` is used in the Multiple component to indicate that more than one option can be selected.
+- `Title` is used to group ListItems together and help visually organize groups.
+- `Separator` is used to create a visual separation between groups.
+
+![Showing the different list types](/assets/components/form/super-select/super-select-list-item-types.png)
+
+## Groups
+
+ListItems can be grouped visually by adding a title and a separator. The last group in the list doesn’t require a separator.
+
+![Groups of list items represented in both Single and Multiple components](/assets/components/form/super-select/super-select-group-list-items.png =836x*)
+
+## Required and optional
+
+For complex forms, required fields should be indicated. This is the most explicit and transparent method to ensure users don’t have to make assumptions. Read more about best practices for [marking required fields in forms](https://www.nngroup.com/articles/required-fields/).
+
+![Showing required next to the label for the Super Select](/assets/components/form/super-select/super-select-required.png )
+
+For shorter, simpler forms (e.g., login/signup and feedback requests), indicate optional fields instead.
+
+![Showing optional next to the label for the Super Select](/assets/components/form/super-select/super-select-optional.png )
+
+## Error validation
+
+For error validation recommendations, refer to the [Form patterns](/patterns/form-patterns?tab=validation) documentation.
+
+## Content
+
+For general content recommendations, refer to the [Primitives](/components/form/primitives) documentation.
+
+### Options list
+
+- Keep option content clear and concise; avoid full sentences.
+- Avoid repeating the same word at the beginning of list options.
+- Order the options in a logical way based on the use case. For example:
+  - default or most commonly selected options first
+  - alphabetically
+  - numerically
```

Comment:
Second-level list items need 4 spaces, not 2, or they will not be rendered as one would expect:
<img width="586" alt="screenshot_3729" src="https://github.com/hashicorp/design-system/assets/686239/3e4a47ce-f87a-4359-a188-edb651f3974b">

```suggestion
- Keep option content clear and concise; avoid full sentences.
- Avoid repeating the same word at the beginning of list options.
- Order the options in a logical way based on the use case. For example:
    - default or most commonly selected options first
    - alphabetically
    - numerically
```